types/debug/index.d.ts

```diff
@@ -30,6 +30,13 @@ declare namespace debug {
         skips: RegExp[];
 
         formatters: Formatters;
+
+        inspectOpts?: {
```

Comment:
These options are only available in `node`, so they are made optional here since they may not exist if running in the browser